app/middlewares/user_validation.js

```diff
@@ -0,0 +1,45 @@
+const { checkSchema, validationResult } = require('express-validator');
+
+const error = require('../errors');
+
+const { userCreationError } = error;
+
+exports.userValidationRules = () => [
+  checkSchema({
+    first_name: {
+      notEmpty: true,
+      errorMessage: "firstName can't be blank"
+    },
+    last_name: {
+      notEmpty: true,
+      errorMessage: "lastName can't be blank"
+    },
+    email: {
+      notEmpty: true,
+      matches: {
+        options: /@(wolox)\.com\.ar$/i,
+        errorMessage: 'you may only use email addresses from wolox domain'
+      }
+    },
+    password: {
+      isLength: {
+        errorMessage: 'Password should be at least 8 characters long',
+        options: { min: 8 }
+      },
+      isAlphanumeric: {
+        errorMessage: 'Password should be alphanumeric'
```

Comment:
Maybe you can create a file for errors messages? And create a folder `/schemas` with your schemas. What do you think?